# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: gfx/wr/wrench/reftests/filters/blend-clipped.png
# Commit: 4b7de1b38a00
# Full Hash: 4b7de1b38a008493b6c6fbb599049358f707e08f
# Author: Nicolas Silva <nsilva@mozilla.com>
# Date: 2021-04-27 15:52:11
# Description:
#   WIP: Bug 1707181 - Avoid zero-sized gradient tasks. r=gfx-reviewers,lsalzman
#   
#   Add missing check for linear gradients. Also, instead of just casting to int round out the task size.
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D113289
# ==============================================================================

diff -r ee55297f011c -r 4b7de1b38a00 gfx/wr/wrench/reftests/filters/blend-clipped.png
Binary file gfx/wr/wrench/reftests/filters/blend-clipped.png has changed
